FWIO/RNTuple/plugins/RNTupleSource.cc

```diff
@@ -304,7 +328,7 @@ namespace edm {
     auto& reader = runReaders_[runPrincipal.index()];
     reader.setEntry(entry);
     runPrincipal.fillRunPrincipal(processHistoryRegistry(), &reader);
-    runPrincipal.setShouldWriteRun(RunPrincipal::kNo);
+    //runPrincipal.setShouldWriteRun(RunPrincipal::kNo);
```

Comment:
Why is this line commented out?

Reply:
Kind of as a reminder that more logic will eventually have to go there to handle case where a Run is supposed to be skipped.

Reply:
Could you add a short comment about it?

Reply:
done